temba/channels/types/discord/type.py

```diff
@@ -0,0 +1,50 @@
+import telegram
+
+from django.urls import reverse
+from django.utils.translation import ugettext_lazy as _
+
+from temba.contacts.models import DISCORD_SCHEME
+
+from ...models import ChannelType
+from .views import ClaimView
+
+
+class DiscordType(ChannelType):
+    """
+    A Telegram bot channel
```

Comment:
This should be Discord not Telegram

Reply:
This is on a pretty old commit. The changeset could be squashed if we want to keep history clean.

Reply:
I think we should be careful when commit squashing (and rebasing). I prefer to keep it honest but historically messy so as to not lose critical history.